packages/theming/src/ThemeProvider/ThemeProvider.js

```diff
@@ -8,13 +8,14 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import { ThemeProvider as StyledThemeProvider } from 'styled-components';
+import merge from 'merge-anything';
```

Comment:
Rather than having "magic" merges, would it be best to require users to spread the theme? An RTL example

```jsx
<ThemeProvider theme={{
    ...DEFAULT_THEME,
    rtl: true
}}>
   ...
</ThemeProvider>
```

Reply:
I'm not a huge fan of how this is turning out as I personally feel the `merge` is quite nice for the consumer. However, I'll keep pushing the changes through as it does simplify Garden's surface area. But we'll need to keep an ear out for how folks think this is affecting their use of the API.